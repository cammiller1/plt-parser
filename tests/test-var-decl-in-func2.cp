## ensure string variables can be declared inside of a function ##

def string bar(int a, int b) { 
	string s;
	s = "I was declared and assigned in the functon";
	return s;
}

string a;
a = bar(1, 2);
print(a);