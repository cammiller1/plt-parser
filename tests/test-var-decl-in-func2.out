I was declared and assigned in the functon